gui/wxpython/startup/guiutils.py

```diff
@@ -692,6 +695,53 @@ def delete_grassdb_interactively(guiparent, grassdb):
     return deleted
 
 
+def can_switch_mapset_interactively(guiparent, grassdb, location, mapset):
+    """
+    Checks if mapset is indicated by the presence of the lock and offers
+    the option to force removal of the lock and to switch to the mapset.
+
+    Returns True if user wants to switch to the selected mapset in spite of
+    removing lock. Returns False if a user wants to stay in the current
+    mapset or if an error was encountered.
+    """
+    can_switch = True
+
+    user = get_current_user()
+    lockfile = get_lockfile_if_present(grassdb, location, mapset)
+    if lockfile:
+        dlg = wx.MessageDialog(
+            parent=guiparent,
+            message=_("User {0} is already running GRASS in selected mapset"
+                      "<{1}>\n (file {2} found).\n\n"
```

Comment:
Using the `{1}` style is sub-optimal. The `{name}` is easier to read, less prone to errors, and more flexible for translations.